docs/rules/bugs/rule-assigns-default.md

```diff
@@ -0,0 +1,61 @@
+# rule-assigns-default
+
+**Summary**: Rule assigned its default value
+
+**Category**: Bugs
+
+**Avoid**
+```rego
+package policy
+
+import rego.v1
+
+default allow := false
+
+# this rule assigns the same value as the default
+# and the policy would work the same without it
+allow := false if {
+    not "admin" in input.user.roles
+}
+```
+
+**Prefer**
+```rego
+package policy
+
+import rego.v1
+
+default allow := false
+
+# or just `allow if {` as `true` is implicit
```

Comment:
I wonder if a non boolean example would be a good addition here too? E.g.

```
default cpu_shares := 0.1

cpu_shares := 0.5 if input.role == "developer"
cpu_shares := 0.1 if input.role == "intern"
```

Or something similar?